docs/reference/attributes.rst

```diff
@@ -1,5 +1,5 @@
 ===========
-Environment
+Attributes
 ===========
```

Comment:
I believe you also need to change number of `=` characters to the length of section name (`Attributes`). Else the document will not build. 

Refer: https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings

Reply:
Oops, didn't know this before. Thanks for pointing it out!